dataland-frontend/src/main.ts

```diff
@@ -7,12 +7,17 @@ import 'material-icons/iconfont/material-icons.css'; // prefer these icons in mo
 import 'primeflex/primeflex.css'
 import './assets/theme/theme.css'
 import '@formkit/themes/genesis'
-import { plugin, defaultConfig } from '@formkit/vue'
+import {plugin, defaultConfig} from '@formkit/vue'
 import router from './router'
 import PrimeVue from 'primevue/config';
 
-const app = createApp(App)
-app.use(plugin, defaultConfig)
-app.use(router)
-app.use(PrimeVue)
-app.mount('#app')
\ No newline at end of file
+function instantiateVueApp() {
```

Comment:
Was ist der Vorteil die Initialisierung hier in eine Funktion zu stecken?

Reply:
Ich könnte mir vorstellen dass es für die Lesbarkeit gedacht ist (was muss alles für die Instantiierung der Vue App getan werden? => logischer "Container").
Ansonsten dürfte es funktional keinen Unterschied machen.